.github/workflows/github-actions-test-installer.yml

```diff
@@ -0,0 +1,46 @@
+name: Test DependencyInstaller script
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
```

Comment:
You can keep pull_request as a trigger for tests, but I think we will run this action on a cron job weekly since the dependencies are stable and change rarely.